packages/plugin-hardhat/src/utils/options.ts

```diff
@@ -50,16 +50,16 @@ type Initializer = {
 };
 
 /**
- * Option to enable or disable Platform deployments.
+ * Option to enable or disable Defender deployments.
  */
-export type Platform = {
-  usePlatformDeploy?: boolean;
+export type DefenderDeploy = {
+  useDefenderDeploy?: boolean;
```

Comment:
I suppose we are doing this change in a minor release despite it being breaking because we documented the feature as "OpenZeppelin Platform is in beta and the functionality described here is subject to change"?

Sounds ok, just want to confirm.